boot/zephyr/keys.c

```diff
@@ -61,14 +61,14 @@ const unsigned char root_pub_der[] = {
     0x8b, 0x68, 0x34, 0xcc, 0x3a, 0x6a, 0xfc, 0x53,
     0x8e, 0xfa, 0xc1, };
 const unsigned int root_pub_der_len = 91;
-#else
-#error "No public key available for given signing algorithm."
```

Comment:
Ah, I see. mcuboot built on Zephyr is never supposed to be built without signing enabled. While testing I built without signing and had to remove this.